content/docs/06-integrations/00-kubernetes.md

```diff
@@ -16,118 +16,589 @@ import Tooltip from "shared/components/ui/Tooltip";
+
+Below are the stages of deprecation:
+
+<br />
 
-## Supported Kubernetes versions
+- **Deprecate**: When a pack is deprecated, Palette displays a warning. You will still be able to create new cluster profiles using the pack and launch clusters using existing profiles that contain the pack.
+
+
+- **Delete**: When a pack is disabled, you will no longer be allowed to create new cluster profiles that using the pack. Palette displays a message informing you to cut over to the newer version of the pack. However, you can still deploy new clusters using existing profiles that contain the deprecated pack.
+
+
+- **Delete**: When a pack is deleted, you will not be able to create new cluster profiles using the pack or launch new clusters using the existing profiles that contain the pack.  Palette displays a message informing you to cut over to the newer version of the pack. There is no disruption to existing cluster workloads and will continue to be functional.
+
+<br />
 
 <InfoBox>
 
-Versions supported in the latest [release](/release-notes/) are highlighted.
+For important guidelines on updating packs, review the [Update Packs](/integrations/update-packs/update-packs) guide.
+
 
 </InfoBox>
 
+
+# Versions Supported
+
+
 <Tabs>
 
-<Tabs.TabPane tab="1.19.x" key="k8s_v1.19">
-
-- **1.19.16**
-- **1.19.15**
-- **1.19.14**
-- **1.19.13**
-- **1.19.12**
-- **1.19.11**
-- **1.19.10**
-- **1.19.9**
-- **1.19.8**
-- **1.19.7**
-- **1.19.6**
-- **1.19.5**
-- **1.19.4**
-- **1.19.3**
+<Tabs.TabPane tab="1.26.x" key="k8s_v1.26">
 
-</Tabs.TabPane>
+## Prerequisites
 
-<Tabs.TabPane tab="1.20.x" key="k8s_v1.20">
-
-- **1.20.12**
-- **1.20.11**
-- **1.20.10**
-- **1.20.9**
-- **1.20.8**
-- **1.20.7**
-- **1.20.6**
-- **1.20.5**
-- **1.20.4**
-- **1.20.2**
-- **1.20.1**
-- **1.20.0**
+- A minimum of 4 CPU and 4GB Memory.
 
-</Tabs.TabPane>
+- Operating System (OS) dependencies as listed in the table.
 
-<Tabs.TabPane tab="1.21.x" key="k8s_v1.21">
+| OS Distribution | OS Version | Supports Kubernetes 1.26.x |
+|---------------|------------|----------------------------|
+| CentOS        | 7.7        | ❌                         |
+| Ubuntu        | 22.04      | ✅                         |
+| Ubuntu        | 20.04      | ❌                         |
+| Ubuntu        | 18.04      | ❌                         |
 
-- **1.21.10**
-- **1.21.6**
-- **1.21.5**
-- **1.21.4**
-- **1.21.3**
-- **1.21.2**
-- **1.21.1**
-- **1.21.0**
 
-</Tabs.TabPane>
+## Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| ``pack:k8sHardening``  | Flag to decide if Kubernetes hardening should be applied. Default: ``True``. When set to ``True``, additional flags configured in `kubeadmconfig` will be honored and will be set to the corresponding components.|
```

Comment:
It turns out that this flag **cannot** be set to `False` as that breaks most cluster deployments. @sadysnaat I thought this flag was going to made hidden?